packages/manager/modules/mfa-enrollment/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@ovh-ux/mfa-enrollment",
+  "version": "0.0.0",
+  "private": true,
+  "license": "BSD-3-Clause",
+  "main": "./src/index.js",
+  "dependencies": {
+    "@ovh-ux/ng-translate-async-loader": "^2.0.2",
```

Comment:
I'm not sure that both following dependencies are needed:

- "@ovh-ux/ng-translate-async-loader": "^2.0.2",
- "bootstrap": "^4.3.1",

Because I can't see any import.